api.js

```diff
@@ -25,6 +25,9 @@ function detailsToVisit(details) {
 }
 
 exports.init = function (sbot, config) {
+  if (!config) config = {}
```

Comment:
`<style>` - any particular reason you don't like?

```
exports.init = function (sbot, config = {}) {
```
Just want to know in case there is a hole I might not know about that I'm gonna fall in because I do it this way all the time

Reply:
No particular reason, JS often has different ways of achieving stuff